recipes/libignition-rendering/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+macos_min_version:             # [osx]
```

Comment:
I am going to ask on the conda-forge gitter if we can merge with the failing build or if we need to skip OS X first.